editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5504,21 +5504,28 @@ void CanvasItemEditorViewport::_create_preview(const Vector<String> &files) cons
 		Ref<Texture2D> texture = Ref<Texture2D>(Object::cast_to<Texture2D>(*res));
 		Ref<PackedScene> scene = Ref<PackedScene>(Object::cast_to<PackedScene>(*res));
 		if (texture != nullptr || scene != nullptr) {
+			bool root_node_selected = EditorNode::get_singleton()->get_editor_selection()->is_selected(EditorNode::get_singleton()->get_edited_scene());
+			String desc = "Drag and drop to add as child of current scene's root node.\nHold Ctrl when dropping to add as child of selected node.\n";
+			if (!root_node_selected) {
+				desc += "Hold Shift when dropping to add as sibling of selected node.\n";
+			}
 			if (texture != nullptr) {
 				Sprite2D *sprite = memnew(Sprite2D);
 				sprite->set_texture(texture);
 				sprite->set_modulate(Color(1, 1, 1, 0.7f));
 				preview_node->add_child(sprite);
 				label->show();
 				label_desc->show();
-				label_desc->set_text(TTR("Drag and drop to add as child of current scene's root node.\nHold Ctrl when dropping to add as child of selected node.\nHold Shift when dropping to add as sibling of selected node.\nHold Alt when dropping to add as a different node type."));
+				desc += "Hold Alt when dropping to add as a different node type\n";
+				label_desc->set_text(TTR(desc));
+
```

Comment:
```suggestion
				desc += "\n" + TTR("Hold Alt when dropping to add as a different node type.");
				label_desc->set_text(desc);
```

Note: Adding missing dot and removed trailing newline in string.